src/System.Security.Cryptography.Xml/src/Configurations.props

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
-      uap;
+      netstandard
```

Comment:
I thought we wanted the `ref` to be netstandard, but the impl to be `netfx` (facade) and `netcoreapp` (real impl) -- which would also prevent losing whatever DSA support we're losing.  Or maybe we need to be netfx+netstandard.  But we definitely need the netfx facade.